appveyor.yml

```diff
@@ -1,14 +1,17 @@
 version: '{build}'
 os: Windows Server 2012
 install:
+  - cmd: choco install zulu8 -ia "INSTALLDIR=""C:\zulu"""
```

Comment:
Appveyor was still working I guess. Is this necessary to do?

Reply:
At AppVeyor, it used Oracle JDK, so I think it would be good to use the same JDK at different OS